hostprocess/eventflow-logger/eventflow-logger.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Does Windows Server 2019 have this runtime installed by default?

Reply:
The build is bundling the runtime into the executable with the `-r win-x64`

https://docs.microsoft.com/en-us/dotnet/core/deploying/#publish-self-contained

Reply:
thanks! this is new to me